stack/startup.sh

```diff
@@ -26,4 +26,6 @@ if [ "x$MANAGEPY_IMPORT_GEOJSON" = "xon" ]; then
 	/venv/bin/python manage.py import_geojson ./geojson/*.json
 	echo "Imported geojson for relation_ids $OSM_RELATION_IDS"
 fi
+
+echo "=> Starting Supervisord"
```

Comment:
You probably don't want this because it'll echo even if we're not running supervisor (which happens when the command is overridden).